src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -195,6 +196,12 @@ public string UserProvidedRegion
 #pragma warning restore CA1305 // Specify IFormatProvider
         }
 
+        public string FallBackToGlobal
```

Comment:
>FallBackToGlobal [](start = 22, length = 16)

why is this not following the pattern of lower casing?

Reply:
I am not entirely sure. This entire file seems to be following this pattern so I used what is already established here.